### PR TITLE
Booking test

### PR DESCRIPTION
**Pull request by itsJoel59**
Added 2 tests, testing the booking modal, using Cypress.

See kanban-card in Trello for a video on the tests running. (File is too big to upload here.)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,5 +12,8 @@
   "license": "ISC",
   "dependencies": {
     "sass": "^1.80.7"
+  },
+  "devDependencies": {
```
Comment (Paajt) [inline package.json:16]: I found a line of code I understand! 😆

**Review by Paajt — APPROVED** [review]
Great job, nice to have a unit test implemented into this project! 👍 💯

**Review by alfredbrannare — APPROVED** [review]
I tried reading through it and it seems like it does the job!

Approved by me!

**Review by Emilyempa — APPROVED** [review]
Nice work. Approved by me

**Pull request merged**